File: YOUR_ERP_CORE/frontend/static/js/safety.js
const SAFETY_STATE = {
    folders: [],
    filtered: [],
    lookups: { leads: [], employees: [], service_profiles: [] },
};

document.addEventListener('DOMContentLoaded', async () => {
    if (!API.requireAuth()) return;
    highlightNav('/app/safety');
    await loadSafetyLookups();
    await loadSafetyFolders();
});

async function loadSafetyLookups() {
    const res = await API.get('/safety/lookups');
    if (!res || res.success === false) {
        showToast((res && res.errors && res.errors[0]) || 'No se pudieron cargar los catalogos.', 'error');
        return;
    }
    SAFETY_STATE.lookups = res.data || SAFETY_STATE.lookups;
    fillFolderModalSelects();
}

async function loadSafetyFolders() {
    const tbody = document.getElementById('safety-folders-tbody');
    if (tbody) tbody.innerHTML = '<tr><td colspan="8" class="empty">Cargando carpetas...</td></tr>';

    const res = await API.get('/safety/folders');
    if (!res || res.success === false) {
        if (tbody) tbody.innerHTML = '<tr><td colspan="8" class="empty" style="color:#ef4444;">Error al cargar carpetas.</td></tr>';
        return;
    }
    SAFETY_STATE.folders = res.data?.results || [];
    applyFolderFilters();
}

function fillFolderModalSelects() {
    const leadSel = document.getElementById('folder-lead');
    const profileSel = document.getElementById('folder-profile');
    const empSel = document.getElementById('folder-assigned-employees');
    const siteSel = document.getElementById('folder-client-site');
    const areaSel = document.getElementById('folder-client-area');
    if (leadSel) {
        const options = ['<option value="">Selecciona una oportunidad</option>'];
        (SAFETY_STATE.lookups.leads || []).forEach((lead) => {
            const label = [lead.project_code || 'PRJ', lead.title, lead.customer_name].filter(Boolean).join(' - ');
            options.push(`<option value="${lead.id}">${esc(label)}</option>`);
        });
        leadSel.innerHTML = options.join('');
        leadSel.onchange = () => {
            refreshFolderSiteOptions();
            refreshFolderAreaOptions();
        };
    }
    if (profileSel) {
        const options = ['<option value="">Automatico segun oportunidad</option>'];
        (SAFETY_STATE.lookups.service_profiles || []).forEach((profile) => {
            options.push(`<option value="${profile.id}">${esc(profile.name)}</option>`);
        });
        profileSel.innerHTML = options.join('');
    }
    if (empSel) {
        empSel.innerHTML = (SAFETY_STATE.lookups.employees || [])
            .map((employee) => `<option value="${employee.id}">${esc(employee.full_name || employee.name || 'Trabajador')}</option>`)
            .join('');
    }
    if (siteSel) {
        siteSel.onchange = () => refreshFolderAreaOptions();
    }
    refreshFolderSiteOptions();
    refreshFolderAreaOptions();
}

function selectedLead() {
    const leadId = Number(document.getElementById('folder-lead')?.value || 0);
    return (SAFETY_STATE.lookups.leads || []).find((lead) => Number(lead.id) === leadId) || null;
}

function refreshFolderSiteOptions() {
    const siteSel = document.getElementById('folder-client-site');
    if (!siteSel) return;
    const lead = selectedLead();
    const customerId = Number(lead?.customer_id || 0);
    const options = ['<option value="">Sin instalacion especifica</option>'];
    (SAFETY_STATE.lookups.client_sites || [])
        .filter((site) => !customerId || Number(site.customer_id) === customerId)
        .forEach((site) => {
            options.push(`<option value="${site.id}">${esc(site.name)}</option>`);
        });
    siteSel.innerHTML = options.join('');
}

function refreshFolderAreaOptions() {
    const areaSel = document.getElementById('folder-client-area');
    const siteSel = document.getElementById('folder-client-site');
    if (!areaSel) return;
    const siteId = Number(siteSel?.value || 0);
    const options = ['<option value="">Sin area especifica</option>'];
    (SAFETY_STATE.lookups.client_areas || [])
        .filter((area) => !siteId || Number(area.site_id) === siteId)
        .forEach((area) => {
            options.push(`<option value="${area.id}">${esc(area.name)}</option>`);
        });
    areaSel.innerHTML = options.join('');
}

function applyFolderFilters() {
    const search = (document.getElementById('filter-search')?.value || '').trim().toLowerCase();
    const traffic = (document.getElementById('filter-traffic')?.value || '').trim().toLowerCase();
    const status = (document.getElementById('filter-status')?.value || '').trim().toLowerCase();
    SAFETY_STATE.filtered = SAFETY_STATE.folders.filter((folder) => {
        const haystack = [
            folder.project_code,
            folder.lead_title,
            folder.customer_name,
            folder.service_profile_name,
        ].join(' ').toLowerCase();
        if (search && !haystack.includes(search)) return false;
        if (traffic && folder.traffic_light !== traffic) return false;
        if (status && String(folder.status || '').toLowerCase() !== status) return false;
        return true;
    });
    renderFolderStats();
    renderFolderTable();
}

function renderFolderStats() {
    const folders = SAFETY_STATE.filtered;
    const green = folders.filter((item) => item.traffic_light === 'green').length;
    const yellow = folders.filter((item) => item.traffic_light === 'yellow').length;
    const red = folders.filter((item) => item.traffic_light === 'red').length;
    setText('stat-total-folders', String(folders.length));
    setText('stat-green', String(green));
    setText('stat-yellow', String(yellow));
    setText('stat-red', String(red));
    setText('stat-total-sub', folders.length ? 'Carpetas visibles segun filtros' : 'Sin carpetas visibles');
}

function renderFolderTable() {
    const tbody = document.getElementById('safety-folders-tbody');
    if (!tbody) return;
    if (!SAFETY_STATE.filtered.length) {
        tbody.innerHTML = '<tr><td colspan="8" class="empty">No hay carpetas para mostrar.</td></tr>';
        return;
    }
    tbody.innerHTML = SAFETY_STATE.filtered.map((folder) => {
        const summary = folder.summary || {};
        return `
            <tr>
                <td>
                    <div style="font-weight:700;color:#f8fafc;">${esc(folder.project_code || 'SIN-CODIGO')}</div>
                    <div style="color:#94a3b8;font-size:0.82rem;">${esc(folder.lead_title || 'Sin oportunidad')}</div>
                </td>
                <td>${esc(folder.customer_name || '-')}</td>
                <td>${esc(folder.service_profile_name || 'Automatico')}</td>
                <td>${trafficChip(folder.traffic_light)}</td>
                <td>
                    <div class="readiness-wrap">
                        <div class="readiness-bar"><span style="width:${Math.max(0, Math.min(100, Number(folder.readiness_pct || 0)))}%;"></span></div>
                        <div class="readiness-label">${Number(folder.readiness_pct || 0).toFixed(1)}% · ${summary.critical_blockers?.length || 0} bloqueos</div>
                    </div>
                </td>
                <td>${esc(folder.planned_start_date || '-')}</td>
                <td>${esc(folder.status || 'draft')}</td>
                <td><a class="btn btn-ghost btn-sm safety-open-btn" href="/app/safety/folders/${folder.id}">Abrir</a></td>
            </tr>
        `;
    }).join('');
}

function trafficChip(color) {
    const labels = { green: 'Verde', yellow: 'Amarillo', red: 'Rojo' };
    const current = color || 'red';
    return `<span class="safety-chip ${current}">${esc(labels[current] || current)}</span>`;
}

function openFolderModal() {
    fillFolderModalSelects();
    const modal = document.getElementById('folder-modal');
    if (modal) modal.classList.add('open');
}

function closeFolderModal() {
    const modal = document.getElementById('folder-modal');
    if (modal) modal.classList.remove('open');
    const lead = document.getElementById('folder-lead');
    const profile = document.getElementById('folder-profile');
    const notes = document.getElementById('folder-notes');
    const date = document.getElementById('folder-planned-date');
    const employees = document.getElementById('folder-assigned-employees');
    if (lead) lead.value = '';
    if (profile) profile.value = '';
    if (notes) notes.value = '';
    if (date) date.value = '';
    const miperNotes = document.getElementById('folder-miper-scope-notes');
    const site = document.getElementById('folder-client-site');
    const area = document.getElementById('folder-client-area');
    if (miperNotes) miperNotes.value = '';
    if (site) site.value = '';
    if (area) area.value = '';
    if (employees) Array.from(employees.options).forEach((option) => { option.selected = false; });
}

async function saveFolder(evt) {
    evt.preventDefault();
    const btn = document.getElementById('save-folder-btn');
    const payload = {
        lead_id: document.getElementById('folder-lead')?.value,
        service_profile_id: document.getElementById('folder-profile')?.value || null,
        planned_start_date: document.getElementById('folder-planned-date')?.value || '',
        client_site_id: document.getElementById('folder-client-site')?.value || null,
        client_area_id: document.getElementById('folder-client-area')?.value || null,
        notes: document.getElementById('folder-notes')?.value || '',
        miper_scope_notes: document.getElementById('folder-miper-scope-notes')?.value || '',
        assigned_employee_ids: selectedValues('folder-assigned-employees'),
    };
    if (!payload.lead_id) {
        showToast('Selecciona una oportunidad.', 'error');
        return;
    }
    if (btn) { btn.disabled = true; btn.textContent = 'Creando...'; }
    const res = await API.post('/safety/folders', payload);
    if (btn) { btn.disabled = false; btn.textContent = 'Crear carpeta'; }
    if (!res || res.success === false) {
        showToast((res && res.errors && res.errors[0]) || 'No se pudo crear la carpeta.', 'error');
        return;
    }
    closeFolderModal();
    showToast('Carpeta creada correctamente.');
    const folderId = res.data?.id;
    if (folderId) {
        window.location.href = '/app/safety/folders/' + folderId;
        return;
    }
    await loadSafetyFolders();
}

function selectedValues(selectId) {
    const select = document.getElementById(selectId);
    if (!select) return [];
    return Array.from(select.selectedOptions).map((option) => option.value).filter(Boolean);
}

function esc(value) {
    return String(value ?? '')
        .replace(/&/g, '&amp;')
        .replace(/</g, '&lt;')
        .replace(/>/g, '&gt;')
        .replace(/"/g, '&quot;');
}

function setText(id, value) {
    const el = document.getElementById(id);
    if (el) el.textContent = value;
}

function statusChip(status) {
    const current = String(status || 'draft').toLowerCase();
    const tone = current === 'ready' ? 'green' : (current === 'in_progress' ? 'yellow' : 'red');
    return `<span class="safety-chip ${tone}">${esc(current)}</span>`;
}

function renderFolderTable() {
    const tbody = document.getElementById('safety-folders-tbody');
    if (!tbody) return;
    if (!SAFETY_STATE.filtered.length) {
        tbody.innerHTML = '<tr><td colspan="8" class="empty">No hay carpetas para mostrar.</td></tr>';
        return;
    }
    tbody.innerHTML = SAFETY_STATE.filtered.map((folder) => {
        const summary = folder.summary || {};
        return `
            <tr>
                <td>
                    <div style="font-weight:700;color:#f8fafc;">${esc(folder.project_code || 'SIN-CODIGO')}</div>
                    <div style="color:#94a3b8;font-size:0.82rem;">${esc(folder.lead_title || 'Sin oportunidad')}</div>
                </td>
                <td>${esc(folder.customer_name || '-')}</td>
                <td>${esc(folder.service_profile_name || 'Automatico')}</td>
                <td>${trafficChip(folder.traffic_light)}</td>
                <td>
                    <div class="readiness-wrap">
                        <div class="readiness-bar"><span style="width:${Math.max(0, Math.min(100, Number(folder.readiness_pct || 0)))}%;"></span></div>
                        <div class="readiness-label">${Number(folder.readiness_pct || 0).toFixed(1)}% - ${summary.critical_blockers?.length || 0} bloqueos</div>
                    </div>
                </td>
                <td>${esc(folder.planned_start_date || '-')}</td>
                <td>${statusChip(folder.status || 'draft')}</td>
                <td><a class="btn btn-ghost btn-sm safety-open-btn" href="/app/safety/folders/${folder.id}">Abrir</a></td>
            </tr>
        `;
    }).join('');
}

File: YOUR_ERP_CORE/frontend/pages/safety.py
from frontend.pages.layout import base_layout


def safety_page():
    content = """
<div id="safety-root" class="workspace-shell">
    <section class="workspace-hero safety-hero">
        <div class="workspace-hero__content">
            <div class="workspace-kicker">MIPER Studio</div>
            <h1>Seguridad y Prevencion</h1>
            <p>
                Gestiona carpetas de arranque, documentos criticos, matriz de riesgo, EPP, charlas y checklists por oportunidad,
                con foco en readiness y trazabilidad.
            </p>
            <div class="safety-hero-chip-row">
                <span class="safety-hero-chip">Carpetas de arranque</span>
                <span class="safety-hero-chip">Generador MIPER</span>
                <span class="safety-hero-chip">Exportacion PDF y Excel</span>
            </div>
        </div>
        <div class="workspace-hero__panel">
            <div>
                <span class="workspace-hero__eyebrow">Acceso rapido</span>
                <div class="workspace-hero__value">Carpetas, matrices y readiness</div>
                <div class="workspace-hero__sub">
                    Crea BOT, procedimientos, matrices y ubicaciones desde un mismo punto de control.
                </div>
            </div>
            <div class="workspace-action-row" style="margin-top:0;">
                <a class="btn btn-secondary" href="/app/safety/activities">Biblioteca BOT</a>
                <a class="btn btn-secondary" href="/app/safety/locations">Ubicaciones MIPER</a>
                <button class="btn btn-primary" onclick="openFolderModal()">+ Nueva carpeta</button>
            </div>
            <div class="workspace-hero__mini-grid">
                <div class="workspace-mini-card">
                    <span>Readiness</span>
                    <strong>Semaforo operativo y avance de arranque</strong>
                </div>
                <div class="workspace-mini-card">
                    <span>Documentos</span>
                    <strong>EPP, charlas, checklists y matriz de riesgo conectados</strong>
                </div>
            </div>
        </div>
    </section>

    <div class="cards-row">
        <div class="stat-card workspace-stat-card accent-blue">
            <div class="label">Total Carpetas</div>
            <div class="value" id="stat-total-folders">0</div>
            <div class="sub" id="stat-total-sub">Sin datos</div>
        </div>
        <div class="stat-card workspace-stat-card accent-emerald">
            <div class="label">Listas</div>
            <div class="value" id="stat-green" style="color:#22c55e;">0</div>
            <div class="sub">Semaforo verde</div>
        </div>
        <div class="stat-card workspace-stat-card accent-amber">
            <div class="label">Con Pendientes</div>
            <div class="value" id="stat-yellow" style="color:#f59e0b;">0</div>
            <div class="sub">Semaforo amarillo</div>
        </div>
        <div class="stat-card workspace-stat-card accent-rose">
            <div class="label">Criticas</div>
            <div class="value" id="stat-red" style="color:#ef4444;">0</div>
            <div class="sub">Semaforo rojo</div>
        </div>
    </div>

    <div class="card workspace-surface safety-filter-card">
        <div class="safety-filter-grid">
            <div class="form-group" style="margin:0;">
                <label>Buscar</label>
                <input type="text" id="filter-search" placeholder="PRJ, oportunidad, cliente o perfil..." oninput="applyFolderFilters()">
            </div>
            <div class="form-group" style="margin:0;">
                <label>Semaforo</label>
                <select id="filter-traffic" onchange="applyFolderFilters()">
                    <option value="">Todos</option>
                    <option value="green">Verde</option>
                    <option value="yellow">Amarillo</option>
                    <option value="red">Rojo</option>
                </select>
            </div>
            <div class="form-group" style="margin:0;">
                <label>Estado</label>
                <select id="filter-status" onchange="applyFolderFilters()">
                    <option value="">Todos</option>
                    <option value="draft">Draft</option>
                    <option value="ready">Ready</option>
                    <option value="in_progress">In Progress</option>
                    <option value="closed">Closed</option>
                </select>
            </div>
        </div>
    </div>

    <div class="card workspace-surface table-card" style="padding:0;overflow:hidden;">
        <table class="data-table" style="width:100%;">
            <thead>
                <tr>
                    <th>Oportunidad</th>
                    <th>Cliente</th>
                    <th>Perfil</th>
                    <th>Procedimiento</th>
                    <th>Semaforo</th>
                    <th>Readiness</th>
                    <th>Arranque</th>
                    <th>Estado</th>
                    <th style="width:100px;">Abrir</th>
                </tr>
            </thead>
            <tbody id="safety-folders-tbody">
                <tr><td colspan="9" class="empty">Cargando carpetas...</td></tr>
            </tbody>
        </table>
    </div>
</div>

<div id="folder-modal" class="modal-overlay">
    <div class="modal miper-builder-modal">
        <div class="miper-builder-head">
            <div>
                <div class="section-kicker">Constructor MIPER</div>
                <h2>Nueva carpeta de seguridad</h2>
                <p>Selecciona la oportunidad y combina procedimientos, cargos, sectores y equipos. La matriz se arma sola desde esas fuentes.</p>
            </div>
            <button type="button" class="btn btn-ghost btn-sm" onclick="closeFolderModal()">Cerrar</button>
        </div>
        <form onsubmit="saveFolder(event)">
            <div class="form-row">
                <div class="form-group">
                    <label>Oportunidad *</label>
                    <select id="folder-lead" required></select>
                </div>
                <div class="form-group">
                    <label>Perfil de servicio</label>
                    <select id="folder-profile"></select>
                </div>
            </div>
            <div class="form-row">
                <div class="form-group">
                    <label>Fecha objetivo de arranque</label>
                    <input type="date" id="folder-planned-date">
                </div>
                <div class="form-group">
                    <label>Instalacion del cliente</label>
                    <select id="folder-client-site"></select>
                </div>
            </div>
            <div class="miper-builder-grid">
                <div class="miper-source-card" data-source-card="procedures">
                    <div class="miper-source-title">
                        <span>Procedimientos</span>
                        <strong id="folder-procedures-count">0</strong>
                    </div>
                    <input type="search" id="folder-procedures-search" placeholder="Buscar PTS, codigo o servicio..." oninput="renderScopeBuilder()">
                    <div id="folder-procedures-selected" class="miper-chip-row"></div>
                    <div id="folder-procedures-list" class="miper-option-list"></div>
                </div>
                <div class="miper-source-card" data-source-card="job_profiles">
                    <div class="miper-source-title">
                        <span>Cargos</span>
                        <strong id="folder-job-profiles-count">0</strong>
                    </div>
                    <input type="search" id="folder-job-profiles-search" placeholder="Buscar cargo o departamento..." oninput="renderScopeBuilder()">
                    <div id="folder-job-profiles-selected" class="miper-chip-row"></div>
                    <div id="folder-job-profiles-list" class="miper-option-list"></div>
                </div>
                <div class="miper-source-card" data-source-card="client_areas">
                    <div class="miper-source-title">
                        <span>Sectores / areas</span>
                        <strong id="folder-client-areas-count">0</strong>
                    </div>
                    <input type="search" id="folder-client-areas-search" placeholder="Buscar area, sector o faena..." oninput="renderScopeBuilder()">
                    <div id="folder-client-areas-selected" class="miper-chip-row"></div>
                    <div id="folder-client-areas-list" class="miper-option-list"></div>
                </div>
                <div class="miper-source-card" data-source-card="equipment_blocks">
                    <div class="miper-source-title">
                        <span>Equipos / herramientas</span>
                        <strong id="folder-equipment-blocks-count">0</strong>
                    </div>
                    <input type="search" id="folder-equipment-blocks-search" placeholder="Buscar equipo, codigo o riesgo..." oninput="renderScopeBuilder()">
                    <div id="folder-equipment-blocks-selected" class="miper-chip-row"></div>
                    <div id="folder-equipment-blocks-list" class="miper-option-list"></div>
                </div>
            </div>
            <div id="folder-scope-preview" class="miper-scope-preview"></div>
            <div class="form-row">
                <div class="form-group">
                    <label>Personal asignado para IRL/EPP/readiness</label>
                    <select id="folder-assigned-employees" multiple size="6"></select>
                    <span class="field-hint">La matriz principal usa cargos/perfiles; el personal queda para IRL, EPP, restricciones y readiness.</span>
                </div>
            </div>
            <div class="form-group">
                <label>Notas iniciales</label>
                <textarea id="folder-notes" rows="4" placeholder="Brechas detectadas, condicion del cliente, alcance del servicio..."></textarea>
            </div>
            <div class="form-group">
                <label>Alcance MIPER</label>
                <textarea id="folder-miper-scope-notes" rows="3" placeholder="Proceso, tareas no rutinarias, interferencias, condiciones del mandante..."></textarea>
            </div>
            <div class="modal-actions">
                <button type="button" class="btn btn-ghost" onclick="closeFolderModal()">Cancelar</button>
                <button type="submit" class="btn btn-primary" id="save-folder-btn">Crear carpeta</button>
            </div>
        </form>
    </div>
</div>

<style>
.safety-hero {
    grid-template-columns:minmax(0,1.2fr) minmax(320px,0.95fr);
}
.safety-hero-chip-row {
    display:flex;
    flex-wrap:wrap;
    gap:0.55rem;
    margin-top:1rem;
}
.safety-hero-chip {
    display:inline-flex;
    align-items:center;
    padding:0.32rem 0.7rem;
    border-radius:999px;
    border:1px solid rgba(148,163,184,0.22);
    background:rgba(15,23,42,0.7);
    color:#e2e8f0;
    font-size:0.75rem;
    font-weight:700;
}
.safety-filter-card {
    background:linear-gradient(180deg, rgba(15,23,42,0.92), rgba(2,6,23,0.88));
}
.safety-filter-grid {
    display:grid;
    grid-template-columns:minmax(240px,2fr) repeat(2,minmax(170px,0.8fr));
    gap:1rem;
    align-items:end;
}
.table-card thead th {
    background:#020617;
    color:#e2e8f0;
}
.safety-chip {
    display:inline-flex;
    align-items:center;
    gap:0.35rem;
    padding:0.28rem 0.65rem;
    border-radius:999px;
    font-size:0.72rem;
    font-weight:700;
    text-transform:uppercase;
    letter-spacing:0.04em;
}
.safety-chip.red { background:#450a0a; color:#fca5a5; border:1px solid #7f1d1d; }
.safety-chip.yellow { background:#422006; color:#fcd34d; border:1px solid #a16207; }
.safety-chip.green { background:#052e16; color:#86efac; border:1px solid #166534; }
.readiness-wrap {
    min-width:170px;
}
.readiness-bar {
    position:relative;
    width:100%;
    height:10px;
    border-radius:999px;
    background:#0f172a;
    border:1px solid #1e293b;
    overflow:hidden;
}
.readiness-bar > span {
    display:block;
    height:100%;
    border-radius:999px;
    background:linear-gradient(90deg,#2563eb 0%,#22c55e 100%);
}
.readiness-label {
    font-size:0.76rem;
    color:#94a3b8;
    margin-top:0.35rem;
}
.safety-open-btn {
    display:inline-flex;
    justify-content:center;
    width:100%;
}
.miper-builder-modal {
    max-width:1180px;
    width:min(1180px, calc(100vw - 2rem));
}
.miper-builder-head {
    display:flex;
    justify-content:space-between;
    gap:1rem;
    align-items:flex-start;
    margin-bottom:1rem;
}
.miper-builder-head p {
    margin:0.35rem 0 0;
    color:#94a3b8;
    max-width:760px;
}
.miper-builder-grid {
    display:grid;
    grid-template-columns:repeat(2,minmax(0,1fr));
    gap:0.9rem;
    margin:1rem 0;
}
.miper-source-card {
    border:1px solid rgba(148,163,184,0.18);
    border-radius:18px;
    background:linear-gradient(180deg, rgba(15,23,42,0.96), rgba(2,6,23,0.88));
    padding:0.9rem;
    min-height:250px;
}
.miper-source-title {
    display:flex;
    justify-content:space-between;
    align-items:center;
    gap:0.75rem;
    margin-bottom:0.7rem;
    color:#f8fafc;
    font-weight:800;
}
.miper-source-title strong {
    min-width:2rem;
    text-align:center;
    border-radius:999px;
    padding:0.16rem 0.55rem;
    background:#1d4ed8;
    color:#fff;
    font-size:0.78rem;
}
.miper-source-card input[type="search"] {
    width:100%;
    margin-bottom:0.65rem;
}
.miper-chip-row {
    display:flex;
    flex-wrap:wrap;
    gap:0.35rem;
    min-height:1.6rem;
    margin-bottom:0.55rem;
}
.miper-chip {
    display:inline-flex;
    align-items:center;
    gap:0.35rem;
    padding:0.24rem 0.52rem;
    border-radius:999px;
    background:rgba(59,130,246,0.15);
    border:1px solid rgba(96,165,250,0.35);
    color:#dbeafe;
    font-size:0.72rem;
    font-weight:700;
}
.miper-chip button {
    border:0;
    background:transparent;
    color:#bfdbfe;
    cursor:pointer;
    font-weight:900;
}
.miper-option-list {
    display:grid;
    gap:0.42rem;
    max-height:190px;
    overflow:auto;
    padding-right:0.2rem;
}
.miper-option {
    width:100%;
    border:1px solid rgba(148,163,184,0.16);
    border-radius:14px;
    background:#0f172a;
    color:#e2e8f0;
    padding:0.58rem 0.7rem;
    cursor:pointer;
    text-align:left;
    transition:border-color .15s ease, background .15s ease, transform .15s ease;
}
.miper-option:hover,
.miper-option.selected {
    border-color:#60a5fa;
    background:#172554;
    transform:translateY(-1px);
}
.miper-option strong {
    display:block;
    color:#f8fafc;
    font-size:0.82rem;
}
.miper-option small {
    display:block;
    margin-top:0.2rem;
    color:#94a3b8;
    font-size:0.72rem;
}
.miper-scope-preview {
    border:1px solid rgba(34,197,94,0.25);
    background:rgba(20,83,45,0.18);
    border-radius:16px;
    padding:0.8rem;
    color:#dcfce7;
    font-size:0.82rem;
    margin-bottom:1rem;
}
@media (max-width: 960px) {
    .safety-filter-grid {
        grid-template-columns:1fr;
    }
    .miper-builder-grid {
        grid-template-columns:1fr;
    }
    #folder-modal .modal {
        width:calc(100% - 1.5rem);
        padding:1.25rem;
    }
}
</style>
"""
    return base_layout("Seguridad", "safety", content, scripts=["safety.js"])
